lottie/src/main/java/com/airbnb/lottie/model/layer/TextLayer.java

```diff
@@ -7,7 +7,9 @@
 import android.graphics.Path;
 import android.graphics.RectF;
 import android.graphics.Typeface;
+import android.util.LruCache;
```

Comment:
I see the inport for LruCache but I don't see any references to it in code.  Did you mean to remove the import when using `SparseArrayCompat` as your cache construct?

Reply:
Yup :)